crates/algokit_utils_ffi/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+use algokit_transact_ffi::{AlgoKitTransactError, Transaction};
+use algokit_utils::Composer as ComposerRs;
+use std::sync::Mutex;
+
+use uniffi::{self};
+
+uniffi::setup_scaffolding!();
+
+#[derive(Debug, thiserror::Error, uniffi::Error)]
+pub enum ComposerError {
+    #[error("TransactionsError: {0}")]
+    TransactionsError(String),
+}
+
+#[derive(uniffi::Object)]
+pub struct Composer {
```

Comment:
Would the idea be to expose all main high level utils abstractions but excluding any user facing methods that would interact with API clients or any excessively stateful logic (deferring the implementation to the target language ffi binding package)? For example how would we implement `send` method on a `TransactionComposer` - i presume it wont be possible to handle it completely on rust side (given extra fact that api clients are handled separately per each target language) but is there a way to expose some sort of abstract stub that can then be implemented in target package (py/ts/...) by ourselves manually?